content/docs/command-reference/studio/logout.md

```diff
@@ -0,0 +1,22 @@
+## studio logout
+
+This command helps to log out user from DVC Studio.
```

Comment:
style: helps -> Log out user from DVC Studio. (simpler is better, no need to use `helps`, etc).